README.md

```diff
@@ -1,7 +1,6 @@
 # JLLWrappers
 
-[![CI](https://github.com/JuliaPackaging/JLLWrappers.jl/workflows/CI/badge.svg)](https://github.com/JuliaPackaging/JLLWrappers.jl/actions?query=workflow%3ACI)
-[![Build Status](https://cloud.drone.io/api/badges/JuliaPackaging/JLLWrappers.jl/status.svg)](https://cloud.drone.io/JuliaPackaging/JLLWrappers.jl)
+[![CI](https://github.com/JuliaPackaging/JLLWrappers.jl/actions/workflows/ci.yml/badge.svg)](https://github.com/JuliaPackaging/JLLWrappers.jl/actions/workflows/ci.yml)
 [![deps](https://juliahub.com/docs/JLLWrappers/deps.svg)](https://juliahub.com/ui/Packages/JLLWrappers/7Zgw7?t=2)
```

Comment:
The JuliaHub link looks broken, too

Reply:
The Dependents are showing fine on README.md.

Reply:
I will update the URL